src/script/api/game/game_window.hpp.sq

```diff
@@ -126,7 +126,7 @@ void SQGSWindow_Register(Squirrel *engine)
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_GAME_OPTIONS,                           "WC_GAME_OPTIONS");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_CUSTOM_CURRENCY,                        "WC_CUSTOM_CURRENCY");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_CHEATS,                                 "WC_CHEATS");
-	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_EXTRA_VIEW_PORT,                        "WC_EXTRA_VIEW_PORT");
+	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_EXTRA_VIEWPORT,                         "WC_EXTRA_VIEWPORT");
```

Comment:
If these are now auto-generated, should they still be under version control?

I tried not editing the file (as suggested by the comment at the top); however, the project failed to build.